### PR TITLE
Changes to support API1600 for LE, EG, SP, SPT and Enclosure

### PR DESCRIPTION
**Pull request by chebroluharika**
### Description
Changes to support API1600 for LE, EG, SP, SPT, Enclosures, Task resources.

### Check List
- [X] New functionality includes testing.
  - [X] All tests pass for go 1.11 + gofmt checks.
- [X] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -115,6 +115,8 @@
 |<sub>/rest/storage-volume-templates</sub>                                                | POST     | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/storage-volume-templates</sub>                                                | DELETE   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/storage-volume-templates</sub>                                                | PUT      | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|     **Tasks**                                                                                                                                     |
```
Comment (AnushaY1916) [inline endpoints-support.md:118]: Task resource is not supported in Terraform SDK, please remove this entry

Comment (chebroluharika) [reply endpoints-support.md:118]: But in endpoints file under vendor folder, there is an entry for Tasks. Do I need to remove there also.

Comment (AnushaY1916) [reply endpoints-support.md:118]: Vendor(Go-lang SDK) has a resource called Task, but in Terraform SDK Task resource doesn't exist. So please add only those endpoints which are supported in Terraform SDK.

Comment (chebroluharika) [reply endpoints-support.md]: okay

**Pull request merged**